src/renderer/components/pool/PoolDetails.tsx

```diff
@@ -43,35 +44,47 @@ export const PoolDetails: React.FC<Props> = ({
   allTimeVolumeTrend,
   totalSwapsTrend,
   totalStakersTrend,
-  isLoading
+  isLoading,
+  HistoryView
 }) => {
+  const renderHistoryView = useMemo(
+    () =>
+      FP.pipe(
+        asset,
```

Comment:
BTW: Why is `asset` an Option in props of `PoolDetail` and not just an `Asset`. If we don't know the asset, we don't have any knowledge about any other data of a pool, which makes `PoolDetail` useless. And in `PoolDetailView` we render `PoolDetails` only if we have an `asset` ...

Reply:
yes. the same thought what came into my head here and i've tried to change it but something went wrong (dont remember what exactly) ([change commit](https://github.com/thorchain/asgardex-electron/pull/1249/commits/f274373c1428907dcde08cfaac50e56f3f0f3e15), and [revert one](https://github.com/thorchain/asgardex-electron/pull/1249/commits/7367c24822c82a33d42a89f5a582a36496b4640e)). anyway now i've tried one more time. works fine